src/test/kotlin/com/ampnet/walletservice/controller/ControllerTestBase.kt

```diff
@@ -207,12 +207,13 @@ abstract class ControllerTestBase : TestBase() {
         amount: Long = 1000,
         type: DepositWithdrawType = DepositWithdrawType.USER,
         txHash: String = "th_ktDw9ytaQ9aSi78qgCAw2JhdzS8F7vGgzYvWeMdRtP6hJnQqG",
-        coop: String = COOP
+        coop: String = COOP,
+        createdBy: UUID = userUuid
     ): Deposit {
         val document = saveFile("doc", "document-link", "type", 1, owner)
         val deposit = Deposit(
             0, owner, "S34SDGFT", amount,
-            ZonedDateTime.now(), userUuid, type, txHash, userUuid,
+            ZonedDateTime.now(), userUuid, type, txHash, createdBy,
```

Comment:
This looks wrong to me. First `userUuid` was for value createdBy and the second was for value approvedBy.
```suggestion
            ZonedDateTime.now(), createdBy, type, txHash, userUuid,
```

If the tests are passing and one test to verify unauthorized confirm deposit.